src/components/Errors.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { useAppContext } from '../context/AppContext';
+
+export const Errors: FC = () => {
+  const { showError, setShowError, errorMessage } = useAppContext();
```

Comment:
i think u can do this without 'showError' and 'setShowError' just display error when error message is not empty or null

Reply:
done ✅
although note that this way the errors are unable to dissapear with a smooth transition